packages/react/CHANGELOG.md

```diff
@@ -6,16 +6,18 @@ This log was last generated on Mon, 21 Jun 2021 07:34:33 GMT and should not be m
 
 ## [8.20.1](https://github.com/microsoft/fluentui/tree/@fluentui/react_v8.20.1)
 
-Mon, 21 Jun 2021 07:34:33 GMT 
```

Comment:
If you can revert this file that would be good (you may need to commit with `-n` to prevent it from being auto-formatted).